src/extension.ts

```diff
@@ -148,16 +155,46 @@ function activateEnvironment(context: vscode.ExtensionContext) {
 
     // register plugin commands
     subscriptions.push(
-        vscode.commands.registerCommand(Commands.CreateCatkinPackage, catkin.createPackage),
-        vscode.commands.registerCommand(Commands.CreateTerminal, utils.createTerminal),
-        vscode.commands.registerCommand(Commands.GetDebugSettings, debug.getDebugSettings),
-        vscode.commands.registerCommand(Commands.ShowCoreStatus, () => { master.launchMonitor(context) }),
-        vscode.commands.registerCommand(Commands.StartRosCore, master.startCore),
-        vscode.commands.registerCommand(Commands.TerminateRosCore, () => { master.stopCore(masterApi) }),
-        vscode.commands.registerCommand(Commands.UpdateCppProperties, build.updateCppProperties),
-        vscode.commands.registerCommand(Commands.UpdatePythonPath, build.updatePythonPath),
-        vscode.commands.registerCommand(Commands.Rosrun, rosrundelegate),
-        vscode.commands.registerCommand(Commands.Roslaunch, roslaunchdelegate),
+        vscode.commands.registerCommand(Commands.CreateCatkinPackage, () => {
+            logger.logCommand(Commands.CreateCatkinPackage);
+            catkin.createPackage();
```

Comment:
Prefer logging within the function rather than having the caller log on its behalf.